gsa/src/web/pages/page.js

```diff
@@ -43,6 +44,7 @@ import CapabilitiesContext from 'web/components/provider/capabilitiesprovider';
 import Footer from 'web/components/structure/footer';
 import Header from 'web/components/structure/header';
 import Main from 'web/components/structure/main';
+import withRouter from 'react-router-dom/withRouter';
```

Comment:
Could you move this import to the top after the react import and change it to the following?
```suggestion
import {withRouter} from 'react-router-dom';
```

Reply:
Just the part about resorting the import is missing :-)
> Could you move this import to the top after the react import